sample_files/.travis.yml

```diff
@@ -0,0 +1,31 @@
+addons:
+    apt:
+        packages:
+            - expect-dev  # provides unbuffer utility
+
+language: python
+
+python:
+    - "2.7"
+
+virtualenv:
+    system_site_packages: true
+
+env:
+    global:
+        - TESTS="0" LINT_CHECK="0" DOCS="0" VERSION="0.1.0" RELEASE="0.1.1" PROJECT="My Project" BRANCH_PROD="master" BRANCH_DOC="gh-pages"
+    matrix:
+        - LINT_CHECK="1"
+        - TESTS="1"
+        - DOCS="1"
+
+install:
+    - git clone --depth=1 https://github.com/LasLabs/python-quality-tools.git -b feature/initial-scripts ${HOME}/python-quality-tools
```

Comment:
This will have to change after it gets pulled to master. I won't delete the branch for now, however.